servicelog/appender/logstash.go

```diff
@@ -78,7 +79,8 @@ func (l *logstash) sendEntry(entry servicelog.Entry) error {
 		return fmt.Errorf("unable to marshal log entry: %s", err)
 	}
 	log.WithField("entry", string(bytes)).Debug("Sending log entry to Logstash")
-	if _, err = l.writer.Write(bytes); err != nil {
+	l.writeTimer.Time(func() { _, err = l.writer.Write(bytes) })
+	if err != nil {
```

Comment:
`err` is always `nil` here, it was checked before. Error handling should be done in the function above.

Reply:
I don't follow, err was checked before - true, but there is another call (wrapped) after that.

Reply:
OK, my bad